dev/bots/prepare_package.dart

```diff
@@ -595,7 +595,7 @@ class ArchivePublisher {
     if (!jsonData.containsKey('current_release')) {
       jsonData['current_release'] = <String, String>{};
     }
-    jsonData['current_release'][branchName] = revision;
+    (jsonData['current_release'] as Map<String, dynamic>)[branchName] = revision;
```

Comment:
Man, JSON really is awkward without dynamic, isn't it?

Would this (and other similar cases) be easier to read if the cast value was broken out into it's own local var?

Reply:
Maybe sometimes?  I tried doing that in some places, but it sometimes just gets in the way by adding more noise.

JSON (and YAML) really suck to use when you have strict typing: they just don't _do_ strict typing.